Core/src/test/java/com/paypal/android/core/HttpRequestFactoryUnitTest.kt

```diff
@@ -1,130 +1,167 @@
 package com.paypal.android.core
 
+import com.paypal.android.core.analytics.AnalyticsEventData
+import com.paypal.android.core.analytics.DeviceInspector
+import org.json.JSONObject
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertNull
 import org.junit.Before
 import org.junit.Test
-import org.junit.experimental.runners.Enclosed
 import org.junit.runner.RunWith
-import org.robolectric.ParameterizedRobolectricTestRunner
 import org.robolectric.RobolectricTestRunner
+import org.skyscreamer.jsonassert.JSONAssert
 import java.net.URL
 
-@RunWith(Enclosed::class)
+@RunWith(RobolectricTestRunner::class)
 class HttpRequestFactoryUnitTest {
 
-    @RunWith(ParameterizedRobolectricTestRunner::class)
```

Comment:
Ended up simplifying this test by removing the parameterized portion.

Reply:
Awesome - thanks for tackling that!